apps/emqx_ft/test/emqx_ft_conf_SUITE.erl

```diff
@@ -299,6 +294,48 @@ t_persist_ssl_certfiles(Config) ->
         emqx_ft_conf:update(mk_storage(true))
     ).
 
+t_import(_Config) ->
+    {ok, _} =
+        emqx_ft_conf:update(
+            mk_storage(true, #{
+                <<"s3">> => mk_s3_config(#{
+                    <<"transport_options">> => #{
+                        <<"ssl">> => #{
+                            <<"certfile">> => emqx_ft_test_helpers:pem_privkey(),
+                            <<"keyfile">> => emqx_ft_test_helpers:pem_privkey()
+                        }
+                    }
+                })
+            })
+        ),
+
+    {ok, #{filename := BackupFile}} = emqx_mgmt_data_backup:export(),
+    {ok, FileNames} = erl_tar:table(BackupFile, [compressed]),
+    [HoconFileName] = lists:filter(
+        fun(N) -> filename:basename(N) =:= "cluster.hocon" end, FileNames
+    ),
+    {ok, [{_, HoconConfig}]} = erl_tar:extract(BackupFile, [
+        memory, compressed, {files, [HoconFileName]}
+    ]),
```

Comment:
Nit. Isn't a bit too much of implementation details leaked here? Maybe it should be the other way around: add ft-related stuff into backup testsuite?

Reply:
The dependency direction is (`emqx_ft` -> depends on -> `emqx_config_backup`), i.e. `emqx_ft` depends on abstraction provided by `emqx_config_backup`. Adding tests to the backup test suite will create a dependency in the opposite direction.

Maybe we should more freely utilize the contract `emqx_config_backup` provides: that it will pass a raw config to `import_config/1` callback, and make this config ourselves for test.

Reply:
I think that as long as there are backup tests that are explicitly integration level tests (i.e. which test backup process with all the stuff that needs backing up) it should be fine. But there's no such testsuite it seems anyway.

> Maybe we should more freely utilize the contract `emqx_config_backup` provides: that it will pass a raw config to `import_config/1` callback, and make this config ourselves for test.

👍🏼

Reply:
I will simplify the test in a followup.